src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -111,30 +111,30 @@ export class ExposureNotificationService {
     this.starting = false;
   }
 
-  async updateSystemStatus(): Promise<SystemStatus> {
+  async updateSystemStatus(): Promise<void> {
     const status = await this.exposureNotification.getStatus();
     this.systemStatus.set(status);
-    return this.systemStatus.get();
   }
 
   async updateExposureStatusInBackground() {
-    const status = await this.updateExposureStatus();
```

Comment:
This is the only use case that we want return value for `updateExposureStatus`. I fix it by calling `this.exposureStatus.get();` after the update.